README.md

```diff
@@ -52,7 +52,7 @@ brew install ec2-instance-selector
 ```
 os=$(uname | tr 'A-Z' 'a-z')
 arch=$(printf "%s" "$(uname -m | tr 'A-Z' 'a-z' | sed -E 's/x86_64|i[3-6]86/amd64/;s/aarch64|arm64/arm64/')")
-curl -Lo ec2-instance-selector https://github.com/aws/amazon-ec2-instance-selector/releases/download/v3.1.0/ec2-instance-selector-$os-$arch && chmod +x ec2-instance-selector
+curl -Lo ec2-instance-selector https://github.com/aws/amazon-ec2-instance-selector/releases/latest/download/ec2-instance-selector-$os-$arch && chmod +x ec2-instance-selector
```

Comment:
Ref: https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases

Reply:
cc: @bwagner5